models/ClimateSensor.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "dtmi:dtdl:context;2",
```

Comment:
In our examples, we should decide on some styling things. For example, do we want @context to be at the top, at the bottom, or in the middle? I personally have usually placed @context at the bottom of the document because it's not very interesting information. I've typically made @id the first element because if you read the doc top down, then you'll have the information about what the doc contains on the first line.

Reply:
@briancr-ms  I placed the context at the top intentionally for 2 reasons:
1. These days I'm dealing with V1 and V2 models at the same time, and hate to scroll to the bottom.
2. In the old XML days, we usually put xmlns in the root element

I'm ok with whatever we decide. There is another question about styling wrt casing of component names. I'm using camelCase.

Reply:
didn't hear back from anyone. Leaving the context at the top for now.